src/Widgets/Common/BackgroundWidget.php

```diff
@@ -37,10 +37,62 @@ public function getSettings()
             ->withOption("inputInterval", 1)
             ->withOption("inputMax", 100);
 
+        $settings->createCheckbox("fullWidth")
+            ->withDefaultValue(true)
+            ->withName("Widget.backgroundFullWidthLabel")
+            ->withTooltip("Widget.backgroundFullWidthTooltip");
+            
+        $settings->createCheckbox("backgroundFixed")
+            ->withDefaultValue(false)
+            ->withName("Widget.backgroundFixedLabel")
+            ->withTooltip("Widget.backgroundFixedTooltip");
+
+        $settings->createCheckbox("backgroundRepeat")
+            ->withCondition("backgroundSize !== 'bg-cover'")
+            ->withDefaultValue(false)
+            ->withName("Widget.backgroundRepeatLabel")
+            ->withTooltip("Widget.backgroundRepeatTooltip");
+
+        $settings->createSelect("backgroundSize")
+            ->withDefaultValue("cover")
```

Comment:
```suggestion
            ->withDefaultValue("bg-cover")
```

Unten heißt der Wert "bg-cover"